packages/databyss-ui/modules/Sidebar/routes/SidebarList.js

```diff
@@ -45,31 +45,32 @@ const menuSvgs = type =>
     topics: <TopicSvg />,
   }[type])
 
-const SidebarList = ({
-  menuItems = defaultMenu,
-  menuItem,
-  menuOpen,
-  //   onToggleMenuOpen,
-  onItemClick,
-}) => {
-  const { getTokensFromPath } = useNavigationContext()
+const SidebarList = ({ menuItems = defaultMenu, menuOpen }) => {
```

Comment:
`SidebarList` is really the root of the Sidebar routes, so let's call it `routes/Root.js`. Why does it need a `menuItems` param?

Reply:
The rendering parts of `SidebarList` should be moved to `components/Sidebar`. Anything specific to loading and compiling the Pages list should be moved to `routes/Pages`. And `defaultMenu` should be moved to `routes/Root`